Syntaxes/Chapel.tmLanguage

```diff
@@ -372,6 +383,44 @@
 			<key>name</key>
 			<string>invalid.illegal.name.chapel</string>
 		</dict>
+		<key>string_escaped_char</key>
+		<dict>
+			<key>patterns</key>
+			<array>
+				<dict>
+					<key>match</key>
+					<string>\\(\\|[abefnprtv'"?]|[0-3]\d{0,2}|[4-7]\d?|x[a-fA-F0-9]{0,2}|u[a-fA-F0-9]{0,4}|U[a-fA-F0-9]{0,8})</string>
+					<key>name</key>
+					<string>constant.character.escape.c</string>
+				</dict>
+				<dict>
+					<key>match</key>
+					<string>\\.</string>
+					<key>name</key>
+					<string>invalid.illegal.unknown-escape.c</string>
+				</dict>
+			</array>
+		</dict>
+		<key>string_placeholder</key>
+		<dict>
```

Comment:
I'm having difficulty determining what the following section is intended to handle.  Can you explain it to me?

Reply:
@lydia-duncan  **[abefnprtv'"?]** this regex represent the simple escape character symbols as pointed by LouisJenkinsCs and **[a-fA-F0-9]{0,2}|u[a-fA-F0-9]{0,4}|U[a-fA-F0-9]{0,8}** regex recognizes all the IPv6 addresses for all the representation described by RFC 2373